x/mongo/driver/topology/polling_srv_records_test.go

```diff
@@ -127,7 +127,7 @@ func compareHosts(t *testing.T, received []description.Server, expected []string
 func TestPollingSRVRecordsSpec(t *testing.T) {
 	for _, tt := range srvPollingTests {
 		t.Run(tt.name, func(t *testing.T) {
-			uri := "mongodb+srv://test1.test.build.10gen.cc/?heartbeatFrequencyMS=100"
+			uri := "mongodb+srv://user:pass@test1.test.build.10gen.cc/?heartbeatFrequencyMS=100"
```

Comment:
Only testing with a URI containing a username/password may cause the same lack of test coverage as there is currently, but for URIs with no username/password.

Is there a way we can test with multiple URI formats? Could we include the URI in the test struct and add test cases for both with and without username/password?